components/Contact.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function ContactPage() {
+  return (
+    <StyledContactArea>
+      <h1>Make Contact</h1>
+      <label>
```

Comment:
Your labels should make use of `htmlFor` in order to be connected to its input field.